R/tidy_add_contrasts.R

```diff
@@ -43,7 +43,7 @@ tidy_add_contrasts <- function(x, model = tidy_get_model(x)) {
       contrasts = NA_character_
     )
     xlevels <- model_get_xlevels(model)
-    for (i in 1:nrow(contrasts_list)) {
+    for (i in seq_len(nrow(contrasts_list))) {
```

Comment:
I ran `goodpractices::goodpractices()` and it suggested we don't use `1:nrow()` notation as it more prone to breaks.